website/src/createSecret/Upload.tsx

```diff
@@ -11,13 +11,13 @@ import {
 import Error from '../shared/Error';
 import Expiration from './../shared/Expiration';
 import Result from '../displaySecret/Result';
-import { randomString, uploadFile } from '../utils/utils';
+import { randomString, uploadFile, getMaxFileSize } from '../utils/utils';
 import { useTranslation } from 'react-i18next';
 import { useForm } from 'react-hook-form';
 import { Grid, Typography } from '@mui/material';
 
 const Upload = () => {
-  const maxSize = 1024 * 500;
```

Comment:
You're right that the maxSize shouldn't be hardcoded here. I'm not sure having a rest API just to get the size is worth the overhead. I think this could be decided at build time and configurable with something like `process.env.FILE_MAX_SIZE`. Alternatively let the client send and reject it server side.

Reply:
How would that work with running the Docker container though? You'd have to build it yourself with the Dockerfile and adjust the FILE_MAX_SIZE env parameter.

The server side rejection sounds more feasible for that scenario imo.

Reply:
> The server side rejection sounds more user friendly for that scenario imo.

If that sounds good to you, I'd go ahead and try to implement it in another PR.